### ISSUE TITLE
Migration Docs for dockerized systems

### ISSUE THREAD

**Issue by mako42** [Graylog2/graylog2-server#18622]
Since many users run Graylog in docker, we might add some migration documentation for those as well.

**Comment by janheise** [comment]
I agree, although this actually has quite some coverage in my [migration.md in the src](https://github.com/Graylog2/graylog2-server/blob/master/data-node/migration/Manual-Migration.md)
